.github/workflows/deploy-dev.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           # Replace input build-folder or version if needed
           build-folder: 'build'
-          graasp-app-id: ${{ secrets.APP_ID }}
+          graasp-app-id: ${{ vars.APP_ID }}
```

Comment:
Is this change due to using a "simple string" as the app id ? So we do not need to "hide" it in secrets ?